src/activation.js

```diff
@@ -180,11 +180,58 @@ function shouldContinue(output, router: Router) {
   return output;
 }
 
+//wraps a subscription, allowing unsubscribe calls even if
+//the first value comes synchronously
+class SafeSubscription {
+  //if this were TypeScript, subscriptionFunc would be of type
+  //(sub: SafeSubscription) => Subscription
```

Comment:
We use flow type annotations. If it's useful information it'd be great to add flow types and remove this comment.